### ISSUE TITLE
Build error with AGP 4.1.0

### ISSUE THREAD

**Issue by davidalbers** [shazam/fork#179]
When upgrading the Android Gradle Plugin from `4.0.1` -> `4.1.0` I get the following error:

```
Cause 1: groovy.lang.MissingPropertyException: Could not get unknown property 'apkNames' for task ':app:packageDebugAndroidTest' of type com.android.build.gradle.tasks.PackageApplication.
	at org.gradle.internal.metaobject.AbstractDynamicObject.getMissingProperty(AbstractDynamicObject.java:85)
	at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:62)
	at com.android.build.gradle.tasks.PackageApplication_Decorated.getProperty(Unknown Source)
	at com.shazam.fork.gradle.ForkPlugin.getApkFileFromPackageAndroidArtifact(ForkPlugin.groovy:112)
	at com.shazam.fork.gradle.ForkPlugin$_createTask_closure3$_closure4.doCall(ForkPlugin.groovy:85)
	at org.gradle.util.ClosureBackedAction.execute(ClosureBackedAction.java:71)
```

Fork version: `3.12.0`

**Comment by davidalbers** [comment]
The `apkNames` function was [removed from AGP](https://android.googlesource.com/platform/tools/base/+/fc01f6bd9bb4a5fd1087f4b36ac46da27c9c4483%5E%21/#F0). I replicated the logic for fork in #180

**Comment by fpezzato** [comment]
Merged in 3.13.0-SNAPSHOT
